src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/JurassicProperties.java

```diff
@@ -17,10 +17,20 @@
 @Validated
 @ConfigurationProperties(prefix = "aws.bedrock.model.jurassic")
 public record JurassicProperties(@Pattern(
+		/**
```

Comment:
The javadoc at the class level... please have each key-value pair appear on a separate line.

Reply:
The hyperlink should updated to be `@see`  with https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-jurassic2.html.